buildspec/linuxIntegrationTests.yml

```diff
@@ -30,6 +30,10 @@ phases:
             # start Docker
             - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay&
             - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+            # Pull SAM related images to avoid delays/timeouts during 'sam local invoke' calls within tests
+            - docker pull lambci/lambda:nodejs8.10
```

Comment:
How are we going to maintain this list? I can see updating the list in integration tests but not here, it passing, then causing random timeouts later. Maybe a comment in the integration tests themselves?

Reply:
I think a comment in the integration tests would work

Reply:
An alternative is to make the calls from somewhere like integrationTest.ts, but I'd prefer to start with a comment